### PR TITLE
Add new ApiContext class

### PR DESCRIPTION
**Pull request by atomatis**
[doc] Add apiContext step and config
[feature] New Api context class
[config] Change composer dependency

### CODE REVIEW

File: src/ApiContext.php [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+    /**
+     * Response data must be json decodable.
+     *
+     * @param string|null $response
+     * @param string|null $responseFileName
+     *
+     * @throws \Exception
+     *
+     * @Then The response body should be empty
```
Comment (stephpy) [inline src/ApiContext.php]: Empty is a good idea, but the 2 other step already exists through:

```
the JSON should be equal to:
"""
...
"""
```

File: src/ApiContext.php [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+     * @param string $headerName
+     * @param string $regex
+     *
+     * @throws \Exception
+     *
+     * @Then The header :headerName should be match with pattern :regex
+     */
+    public function headerShouldBeEqualToRegExp(string $headerName, string $regex = null): void
+    {
+        $headerValue = $this->restApiBrowser->getRequestHeaders()[$headerName] ?? null;
```
[the remark this answers is not in the document]
Comment (stephpy) [reply src/ApiContext.php]: However, It makes sense to have it on response

File: composer.json [hunk 3]
```diff
@@ -21,6 +21,14 @@
...
-        "squizlabs/php_codesniffer": "~3.1.1"
-    }
+        "squizlabs/php_codesniffer": "~3.1.1",
+        "ubirak/rest-api-behat-extension": "^7.0",
+        "swaggest/json-diff": "dev-master"
+    },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "git@github.com:atomatis/json-diff.git"
```
Comment (stephpy) [inline composer.json]: Why do we need to use your fork ?

Comment (atomatis) [reply composer.json]: The component don't have simple tools for get diff value between original and new Json string for a specific path.
Without the fork, i need to add complex parsing method in context, losing most of interest to use this component.

**Pull request closed without merging**